youtrack_sdk/helpers.py

```diff
@@ -114,3 +115,9 @@ def custom_json_dumps(obj: Any) -> str:
 
 def obj_to_json(obj: Optional[BaseModel]) -> str:
     return custom_json_dumps(obj_to_dict(obj))
+
+
+class IssueLinkDirection(enum.Enum):
+    OUTWARD = "s"
+    INWARD = "t"
+    BOTH = ""
```

Comment:
I would put it to `types.py` or `entities.py`. `helpers.py` for me is the place for helper functions, not for types.